src/input/dependency.js

```diff
@@ -17,6 +46,9 @@ const smartGlob = require('../smart_glob.js');
  * @returns results
  */
 function dependencyStream(indexes, config) {
+  const babelConfig = config.babel
+    ? { configFile: path.resolve(__dirname, '../../../../', config.babel) }
```

Comment:
Idea here is to load the config relative to wherever `documentation` was installed.

Reply:
might be easier to manage as an absolute path